topics/data-science/tutorials/python-math/tutorial.md

```diff
@@ -0,0 +1,361 @@
+---
+layout: tutorial_hands_on
+
+title: Python - Math
+level: Introductory
+requirements: []
+follow_up_training: []
+questions:
+- How do I do math in Python?
+
+objectives:
+- Understand the fundamentals of object assignment and math in python and can write simple statements and execute calcualtions in order to be able to summarize the results of calculations and classify valid and invalid statements.
+- Translate some known math functions (e.g. euclidean distance, root algorithm) into python to transfer concepts from mathematics lessons directly into Python.
+
+time_estimation:  30M
+key_points:
+- Converting mathematics equation to python is pretty easy!
+- In real life you'll occasionally need to do this, either converting from a formula you read in a paper, or a description of an algorithm, into code that you'll re-use.
+- Did you forget how a particular module or function works? Try `help()`
+
+enable: false
+subtopic: python-modular
+contributors:
+  - hexylena
+  - dirowa
+
+priority: 1
+notebook:
+  language: python
+---
+
+Here we'll learn some of the fundamentals of python and how to do basic maths in Python.
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+# Python Fundamentals
+
+## Variables
+
+Any Python interpreter can be used as a calculator. We can do simple sums
+
+```python
+3 + 5 - 4
+```
+
+Multiplication
+
+```python
+5 * 4
+```
+
+Division
+
+```python
+5 / 4
+```
+
+Just like you've probably learned in maths courses, we can assign values to variables
+
+```python
+x = 10
+y = 2.5
+z = 3
+```
+
+From now on, whenever we use `x` or `y` or `z`, Python will substitute the value we assigned to it.
+You'll notice when we assign things to a variable, nothing is printed out. If we've assigned a value to a variable we can print it by doing:
+
+```python
+print(x)
+print(y, z)
+```
+
+You can print out multiple variables if you separate them with a comma `,`.
+
+
+> ### {% icon tip %} Tip: Variable Names
+> In Python, variable names:
+>
+> - can include letters, digits, and underscores
+> - cannot start with a digit
+> - are case sensitive
+>
+> This means that, for example:
+>
+> - `x` is a valid name
+> - `weight0` is a valid variable name, whereas `0weight` is not
+> - `weight` and `Weight` are different variables
+{: .tip}
+
+```python
+x * (y + z)
+```
+
+What do you think the following will result in? The best way to find out is trying things for yourself. If it's wrong, can you correct it?
+
+```python
+xy
+```
+
+Time to check what we've learned!
+
+> ### {% icon question %} Exercise 0: Simple Equations
```

Comment:
@hexylena 
Maybe add a line that they should use a python script in the box below

Reply:
Can do!